azure/apis/network/v1alpha2/types.go

```diff
@@ -1,17 +1,16 @@
 /*
 Copyright 2019 The Crossplane Authors.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at
```

Comment:
Looks like license header was changed here but not elsewhere, is that intended?

Reply:
Ah I think this was me trying out that reformat extension you recommended. Will fix.